### PR TITLE
Matrix Transpose(2D Arrays)

### PR DESCRIPTION
**Pull request by sayantani11**
Fixed issue #160 
Please review and suggest if any changes required.
Hoping for a feedback soon.
Thank you for letting me work for this issue.

### CODE REVIEW

File: Topic Wise Questions with Solutions/Array/Matrix_Transpose.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    A=[]   #initialising the matrix
+    print("Enter the elements row wise: ")
+
+    for i in range(N):
+        a=[]
+        for j in range(N):
+            a.append(int(input()))
+        A.append(a)
+
+    # For the matrix transpose:
```
Comment (rohanbabbar04) [inline Topic Wise Questions with Solutions/Array/Matrix_Transpose.py:29]: Define a function for ease of understanding

File: Topic Wise Questions with Solutions/Array/Matrix_Transpose.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Given a square matrix A (n*n) & its number of rows (or columns) N, return the Transpose of A.
+
+# The transpose of a matrix is the matrix flipped over its main diagonal,
+#    switching the row and column indices of the matrix
+
+# Constraints: 1<=N<=1000
+
+# input = 1 2 3
```
Comment (rohanbabbar04) [inline Topic Wise Questions with Solutions/Array/Matrix_Transpose.py:8]: Add the sample output after the code

**Review by rohanbabbar04 — CHANGES_REQUESTED** [review]

**Comment by sayantani11** [comment]
I did define it by a function

**Review by rohanbabbar04 — APPROVED** [review]

**Comment by sayantani11** [comment]
@rcoder23 The changes are approved.......isn't it done?

**Comment by sayantani11** [comment]
@rcoder23  Can you please update me?
The changes are approved....Then why the tag?

**Review by rcoder23 — APPROVED** [review]
Correct

**Pull request merged**